public/api/menu.json

```diff
@@ -0,0 +1,85 @@
+{
+  "items": [
+    {
+      "id": "1",
+      "image": true,
```

Comment:
Why is image a boolean? Maybe you should put here the link to the image?

Reply:
I wanna try to build the url according to the ids myself